src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -131,21 +145,15 @@ def unused_items
         item_for(:unused, _("Unused Devices"), icon: Icons::UNUSED)
       end
 
-      def item_for(id, title, icon: nil, subtree: [])
-        args = [Id(id)]
-        args << term(:icon, icon) if icon
-        args << title
-        args << open?(id)
-        args << subtree
-        Item(*args)
+      def item_for(id, label, widget: nil, icon: nil, subtree: [])
+        contents = widget ? VBox(widget) : Empty()
+        page = CWM::Page.new(widget_id: id, label: label, contents: contents)
+        CWM::PagerTreeItem.new(page,
+          icon: icon, open: open?(id), children: subtree)
       end
 
       def open?(id)
-        @opened.include?(id)
-      end
-
-      def term(*args)
-        Yast::Term.new(*args)
+        id == :all
```

Comment:
in general previous version if you see it, have opened in constructor. So it get list of ids to be opened. In general overview it is `:all` only, but it can be in fact anything. I am fine, if you have it smarter and let widget itself consider what should be opened. I can point you to original storage ep implementation if you are interested.